app/src/main/java/com/github/se/project/ui/lesson/RequestedLessons.kt

```diff
@@ -187,18 +187,6 @@ fun RequestedLessonsScreen(
             selectedItem = navigationActions.currentRoute(),
             networkStatusViewModel = networkStatusViewModel)
       }) { padding ->
-        var refreshing by remember { mutableStateOf(false) }
-        val refreshScope = rememberCoroutineScope()
-
-        fun refresh() =
-            refreshScope.launch {
-              refreshing = true
```

Comment:
Why remove the refesh of the requestLesson? I might not be up to date with the reason of this, some clarifications would be appreciated.

Reply:
This function is defined twice, I kept one version outside of the scafold such that it can be used any where on the requested screen